common/plugins/eu.esdihumboldt.hale.common.headless/src/eu/esdihumboldt/hale/common/headless/transform/Transformation.java

```diff
@@ -457,7 +462,27 @@ public void done(IJobChangeEvent event) {
 			storeJob.schedule();
 		}
 		else {
-			// otherwise schedule jobs directly
+			// otherwise feed InstanceProcessors directly from the
+			// InstanceCollection...
+			final InstanceProcessingExtension ext = new InstanceProcessingExtension(
+					serviceProvider);
+			final List<InstanceProcessor> processors = ext.getInstanceProcessors();
+
+			ResourceIterator<Instance> it = sourceToUse.iterator();
+			try {
+				while (it.hasNext()) {
+					Instance instance = it.next();
+
+					ResolvableInstanceReference resolvableRef = new ResolvableInstanceReference(
+							new PseudoInstanceReference(instance), sourceToUse);
```

Comment:
Having the memory fill with all the instances is a problem. Maybe we should instead introduce a flag for the execution if instance processing is needed (or even which kind is needed) and then use the OrientDB storage.